apps/frontend/src/app/Components/Artifact/ArtifactFilterDisplay.tsx

```diff
@@ -141,9 +171,14 @@ export default function ArtifactFilterDisplay({ filterOption, filterOptionDispat
       <ArtifactSetMultiAutocomplete totals={artSetTotal} artSetKeys={artSetKeys} setArtSetKeys={artSetKeys => filterOptionDispatch({ artSetKeys })} />
       <ArtifactMainStatMultiAutocomplete totals={artMainTotal} mainStatKeys={mainStatKeys} setMainStatKeys={mainStatKeys => filterOptionDispatch({ mainStatKeys })} />
       <ArtifactSubstatMultiAutocomplete totals={artSubTotal} substatKeys={substats} setSubstatKeys={substats => filterOptionDispatch({ substats })} />
-      <LocationFilterMultiAutocomplete locations={showEquipped ? [] : locations} setLocations={locations => filterOptionDispatch({ locations })} disabled={showEquipped} />
-      <Button startIcon={<PersonSearch />} color={showEquipped ? "success" : "secondary"} onClick={() => filterOptionDispatch({ showEquipped: !showEquipped })}>{t`equippedArt`}</Button>
-      <Button startIcon={<BusinessCenter />} color={showInventory ? "success" : "secondary"} onClick={() => filterOptionDispatch({ showInventory: !showInventory })}>{t`artInInv`}</Button>
+      <Suspense fallback={null}>
+        <Tooltip title={t`locationsTooltip`} placement="top" open={showEquipped ? locationsTooltipOpen : false} disableInteractive arrow>
+          <Box onMouseOver={() => setlocationsTooltipOpen(true)} onMouseLeave={() => setlocationsTooltipOpen(false)} >

```

Comment:
You don't need to do all this `Box` with state funkiness. Wrapping the autocomplete in a span is enough, and preserves mobile tooltip functionality:

```
        <BootstrapTooltip title={showEquipped? t`locationsTooltip` : ""} placement="top">
          <span>
            <LocationFilterMultiAutocomplete totals={locationTotal} locations={showEquipped ? [] : locations} setLocations={locations =>
              filterOptionDispatch({ locations })} disabled={showEquipped} />
          </span>
        </BootstrapTooltip>
```

Reply:
The intention here is to only show the tooltip when the location autocomplete is disabled.

Reply:
My code does that. 

`title={showEquipped ? t("locationsTooltip") : ""}`

a blank `title` disables the tooltip automatically

Reply:
huh it didnt work for me, I guess it didnt hotload. I'll make the change, good catch.